utils/build/docker/java/spring-boot.Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM maven:3.9-eclipse-temurin-11 as build
-
-COPY ./utils/build/docker/java/iast-common/src /iast-common/src
+FROM datadog/system-tests:spring-boot.base-v0 as build
 
 WORKDIR /app
 
-COPY ./utils/build/docker/java/spring-boot/pom.xml .
-RUN mkdir /maven && mvn -Dmaven.repo.local=/maven -B dependency:go-offline
-
+COPY ./utils/build/docker/java/iast-common/src /iast-common/src
 COPY ./utils/build/docker/java/spring-boot/src ./src
-RUN mvn -Dmaven.repo.local=/maven package
+
+# TODO: is this line download stuff? if yes, include it in base image
+RUN mvn -Dmaven.repo.local=/maven package  
```

Comment:
Yes, it does. Unfortunately some things are only fetched once the sources are available and all build stages run. But at least what is fetched does not depend on sources' contents except pom.xml, so it would be ok to include them in the base image for build (perhaps deleting them later?).

Reply:
IIUC, we could preload them in the base image, and so this command won't have to download it, but would pick it from the base image? If yes, that's perfect.

Reply:
Yes, I think the main aspect to consider here is not mix old with new sources. I'm not sure what's `COPY` behavior if  you do it over an existing directory. If it fully replaces the directory, then it's fine to include the sources as is and run package in the base image. Otherwise, it's just the same but deleting the sources from base image before finishing.

Reply:
let's zoom when we'll be back !